src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  box-sizing: border-box;
+}
+
 body {
+  --color: #00acdc;
+
+  font-size: 12px;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  background: white;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 50px;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.nav_list {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav_item {
+  font-weight: 500;
+  text-transform: uppercase;
+  text-align: center;
+  margin-right: 50px;
+  margin-left: 20px;
+  font-size: 12px;
+}
+
+.nav_item:first-child {
+  margin-left: 0;
+}
+
+.nav_link {
+  text-decoration: none;
+  color: black;
+  align-items: center;
+  height: 60px;
+  padding: 0;
   margin: 0;
+  display: flex;
+}
+
+.nav_link:hover {
+  color: (var(--color));
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
```

Comment:
```suggestion
  color: var(--color);
```

Reply:
Not fixed 
<img width="909" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/bbc47f4d-c7bc-4971-98d9-c7bc1dca448f">